v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
+    },
+    {
+      "title": "Social Login",
+      "icon": "social",
```

Comment:
Spacing around the images is not as per design

<img width="246" alt="Screenshot 2023-03-13 at 6 48 12 PM" src="https://user-images.githubusercontent.com/18233774/224713589-cefc1969-c740-421f-ad4f-6cec1523d27e.png">